packages/frontend/src/components/common/PivotTable/index.tsx

```diff
@@ -517,7 +690,7 @@ const PivotTable: FC<PivotTableProps> = ({
                             index={totalRowIndex}
                         >
                             {/* shows empty cell if row numbers are visible */}
-                            {hideRowNumbers ? null : <Table.Cell />}
+                            {hideRowNumbers ? null : <Table.CellHead />}
```

Comment:
iirc CellHead renders th instead of td which would be semantically inaccurate. what's the reason of using it here?

Reply:
There was a reason, but I cannot remember. Something to do with formatting. Let me see if I can figure it out.

Reply:
OK, I think it is because the cell in the bottom left renders in white if we use Cell:
<img width="84" alt="image" src="https://github.com/user-attachments/assets/988e2c21-f1d4-4633-a650-ab82eef2e9b9">
With CellHead:
<img width="312" alt="image" src="https://github.com/user-attachments/assets/75ce5170-e2b3-4409-9c64-23feac6f8526">

Reply:
You're right about th instead of td, but the other row total "cells" also render as th:
<img width="217" alt="image" src="https://github.com/user-attachments/assets/33e14bb7-c78d-44ad-85f1-bd2ad728138d">